### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by ovari**

### CODE REVIEW

**Comment by plata** [comment]
I'm not convinced that it's really useful to link wiki pages for Linux/Maven etc.

@madoar @qparis @ImperatorS79 what's your opinion?

**Comment by madoar** [comment]
I tend to agree.

In my opinion some links are quite useful for example the links to:
- POL 4
- Wine
- free software

The other links I think we do not need in the README file.

File: README.md [hunk 1]
```diff
@@ -3,23 +3,41 @@
 [![Codacy Badge](https://api.codacy.com/project/badge/Grade/b667020df53c4b80a22d7e5a73f2b1b3)](https://www.codacy.com/app/PhoenicisOrg/phoenicis?utm_source=github.com&amp;utm_medium=referral&amp;utm_content=PhoenicisOrg/phoenicis&amp;utm_campaign=Badge_Grade)
 [![Crowdin Badge](https://d322cqt584bo4o.cloudfront.net/phoenicis/localized.svg)](https://crowdin.com/project/phoenicis)
 
-Phoenicis is the designated successor of PlayOnLinux and PlayOnMac 4. It allows you to install and use non-native applications on your favorite operating system.
+Phoenicis is the designated successor of [PlayOnLinux and PlayOnMac 4](https://github.com/PlayOnLinux/POL-POM-4). It allows you to install and use non-native applications on your favorite operating system.
 
 Supported operating systems:
-* Linux
-* Mac OSX
+* [GNU/Linux](https://en.wikipedia.org/wiki/Linux)
```
Comment (madoar) [inline README.md:9]:
```suggestion
* GNU/Linux
```

File: README.md [hunk 2]
```diff
@@ -3,23 +3,41 @@
...
+* [macOS](https://en.wikipedia.org/wiki/MacOS)
```
Comment (madoar) [inline README.md:10]:
```suggestion
macOS
```

File: README.md [hunk 3]
```diff
@@ -3,23 +3,41 @@
...
+# What is Phoenicis PlayOnLinux?
+
+Phoenicis PlayOnLinux is a piece of software which allows you to easily install and use numerous games and apps designed to run with Microsoft® Windows®.
+
+Few games are compatible with GNU/Linux at the moment and it certainly is a factor preventing the migration to this system. Phoenicis PlayOnLinux brings a cost-free, accessible and efficient solution to this problem.
+
+# What are Phoenicis PlayOnLinux’s features?
+
+Here is a non-exhaustive list of the interesting points to know:
+* You don’t have to own a [Microsoft Windows](https://en.wikipedia.org/wiki/Microsoft_Windows) license to use PlayOnLinux.
```
Comment (madoar) [inline README.md:26]:
```suggestion
* You don’t have to own a Microsoft Windows license to use PlayOnLinux.
```

File: README.md [hunk 4]
```diff
@@ -3,23 +3,41 @@
...
+* Phoenicis PlayOnLinux is based on [Wine](https://www.winehq.org/), and so profits from all its features yet it keeps the user from having to deal with its complexity.
+* Phoenicis PlayOnLinux is [free software](https://en.wikipedia.org/wiki/Free_software).
+* Phoenicis PlayOnLinux uses [Java](https://en.wikipedia.org/wiki/Java_(programming_language)).
```
Comment (madoar) [inline README.md:29]:
```suggestion
* Phoenicis PlayOnLinux uses Java.
```

File: README.md [hunk 5]
```diff
@@ -3,23 +3,41 @@
...
+* Not all games are supported. Nevertheless, you can use our manual installation module.
+
+For more information, visit http://www.playonlinux.com
 
 ## Build and Run locally
 Dependencies:
-* Java 11+
-* Maven
+* [Java 11+](https://en.wikipedia.org/wiki/Java_(programming_language))
+* [Maven](https://en.wikipedia.org/wiki/Apache_Maven)
```
Comment (madoar) [inline README.md:40]:
```suggestion
* Java 11+
* Maven
```

File: README.md (hunk 1 above)
Comment (ovari) [reply README.md]: Done

File: README.md (hunk 2 above)
Comment (ovari) [reply README.md]: Done

File: README.md (hunk 3 above)
Comment (ovari) [reply README.md]: Done

File: README.md (hunk 4 above)
Comment (ovari) [reply README.md]: Done

File: README.md (hunk 5 above)
Comment (ovari) [reply README.md]: Done

**Comment by ovari** [comment]
Can this merge request please be merged so that we can delete our fork? Thank you

**Pull request merged**